File: Day-03-of-30-Days/Solution.py
# Day 03 of 30 Days 👾

def is_leap_year(year):
    if year % 400 == 0:
        return True
    elif year % 4 == 0 and year % 100 != 0:
        return True
    else:
        return False

# Example
print(is_leap_year(2016))

File: Day-03-of-30-Days/Solution.cpp
// Day 03 of 30 Days 👾

#include <iostream>
using namespace std;

bool isLeapYear(int year) {
    if(year % 400 == 0) {
        return true;
    } else if(year % 4 == 0 && year % 100 != 0) {
        return true;
    } else {
        return false;
    }
}

// examples
int main() {
    cout << isLeapYear(2019) << endl; // return 0 means false
    cout << isLeapYear(2016) << endl; // return 1 means true
}